Sources/protoc-gen-grpc-swift/main.swift

```diff
@@ -169,6 +172,7 @@ func main(args: [String]) throws {
           generatedFiles: &generatedFiles
         )
         if options.v2 {
+          #if compiler(>=6.0)
```

Comment:
Should we `fatalError` (or error in some other way) if the compiler is not Swift 6+ but the v2 option is set?

Reply:
Hmm, good question. I think we should actually put the v2 option behind the compiler guard too tbh.